File: src/main/java/com/f4erp/help_desk/HelpDeskApplication.java
package com.f4erp.help_desk;

import org.camunda.bpm.spring.boot.starter.annotation.EnableProcessApplication;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@EnableProcessApplication("HelpDesk")
public class HelpDeskApplication {
  public static void main(String... args) {
    SpringApplication.run(HelpDeskApplication.class, args);
  }
}

File: src/main/kotlin/com/f4erp/help_desk/HelpDeskApplication.kt
package com.f4erp.help_desk

//import firebase.auth.EnableFirebaseSecurity
import org.camunda.bpm.spring.boot.starter.annotation.EnableProcessApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@SpringBootApplication
@EnableJpaRepositories(basePackages = ["com.f4erp.help_desk.repositories"])
@Configuration
@EnableProcessApplication("HelpDesk")
open class HelpDeskApplication

fun main(args: Array<String>) {
    runApplication<HelpDeskApplication>(*args)
}
